packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -10,14 +17,89 @@ export class UsaBanner extends React.PureComponent {
   constructor(props) {
     super(props);
     this.id = props.id || uniqueId('gov-banner_');
-    this.state = { isBannerOpen: false };
+    this.media = window.matchMedia('(max-width: 543px)');
+    this.state = {
+      isBannerOpen: false,
+      shouldRenderMobileView: this.media.matches,
+    };
     this.handleToggleBanner = this.handleToggleBanner.bind(this);
+    this.onMediaChange = this.onMediaChange.bind(this);
+
+    this.media.addEventListener('change', this.onMediaChange);
+  }
+
+  onMediaChange(evt) {
+    this.setState({ shouldRenderMobileView: evt.matches });
   }
 
   handleToggleBanner() {
     this.setState({ isBannerOpen: !this.state.isBannerOpen });
   }
 
+  // on mobile, the entire header needs to be a clickable element
+  renderMobileHeaderContent(t) {
+    return (
+      <button
+        onClick={this.handleToggleBanner}
+        className="ds-c-usa-banner__button"
+        aria-expanded={this.state.isBannerOpen}
+        aria-controls={this.id}
+      >
+        <p className="ds-c-usa-banner__header-text">
+          <UsaFlagIcon
+            className="ds-c-usa-banner__header-flag"
+            title={this.props.locale === 'es' ? 'U.S. Bandera' : 'U.S. Flag'}
+          />
+        </p>
+        <p className="ds-c-usa-banner__header-text">
+          <span>{t.bannerText}</span>
+          {!this.state.isBannerOpen && (
+            <span className="ds-c-usa-banner__cta-wrapper">
+              <span className="ds-c-usa-banner__button-text">{t.bannerActionText}</span>
+              <ArrowIcon direction="down" className="ds-c-usa-banner__action-icon" />
+            </span>
+          )}
+        </p>
+        {this.state.isBannerOpen && (
+          <div className="ds-c-usa-banner__collapse-banner-container">
+            <CloseIconThin />
+          </div>
+        )}
+      </button>
+    );
+  }
+
+  // on larger screens, only cta needs to be clickable
+  renderHeaderContent(t) {
+    return (
+      <>
+        <p className="ds-c-usa-banner__header-text">
```

Comment:
Again, not sure this `<p>` is needed when a class could be applied to the SVG itself